src/neptune/objects/project.py

```diff
@@ -152,7 +152,8 @@ def __init__(
         async_no_progress_callback: Optional[NeptuneObjectCallback] = None,
         async_no_progress_threshold: float = ASYNC_NO_PROGRESS_THRESHOLD,
     ):
-        raise NeptuneUnsupportedFunctionalityException
+        if mode in {Mode.ASYNC.value, Mode.SYNC.value}:
+            raise NeptuneUnsupportedFunctionalityException
```

Comment:
If I understand correctly we are now only allowing `read-only` and `debug` modes.
I guess it makes sense to disable Project only in cases it has to be connected to backend

Reply:
+ We do not support offline mode for anything except Runs.